### DISCUSSION TITLE
how to get or set the value of ApplicationContext

### DISCUSSION THREAD

**Discussion by designcsla** [MarimerLLC/csla#4140] [Questions]
I'm using a 8.0 Wpf Project. I have a question that how to get or set the ApplicationContext value to use on the whole solution?

**Comment by rockfordlhotka** [comment]
This might be helpful

https://github.com/MarimerLLC/csla/blob/main/docs/Upgrading%20to%20CSLA%206.md#wpf-app

**Reply by designcsla** [reply]
thanks,another question. why the system say: DataPortal Fetch failed on the server?

**Reply by rockfordlhotka** [reply]
If you look at the ToString of the exception object the details are usually in the first couple lines of the output.
